test/elastic-adapter.spec.js

```diff
@@ -138,7 +160,7 @@ describe('elastic source', function() {
                 });
             });
 
-            it('writes a point with a giant field', function() {
+            it('fails to write a point with a giant field', function() {
```

Comment:
It's related! It turns out we were able to write the big string because the analyzer broke it into teeny chunks, but without analysis it tries to store the whole thing which is too much.